core/driver/sources/src/main/scala/org/virtuslab/ideprobe/reporting/ErrorValidator.scala

```diff
@@ -1,13 +1,15 @@
 package org.virtuslab.ideprobe.reporting
 
+import scala.collection.mutable
+
 import org.virtuslab.ideprobe.config.CheckConfig
 import org.virtuslab.ideprobe.protocol.IdeMessage
 
 object ErrorValidator {
   def apply(config: CheckConfig, errors: Seq[IdeMessage]): Option[Exception] = {
     val filteredErrors = errors
-      .filter(error => config.errors.includeMessages.exists(_.r.findFirstIn(error.content).nonEmpty))
-      .filterNot(error => config.errors.excludeMessages.exists(_.r.findFirstIn(error.content).nonEmpty))
+      .filter(error => ideMessageMatchesMessagesFromConfig(error, config.errors.includeMessages))
```

Comment:
Hmmm for clarity... what's the default value for `includeMessages`? if the user doesn't modify the config, will **all** error messages be included by default? 🤔

Reply:
```
includeMessages = [".*"]
excludeMessages = []
```
...
Looks like some update will be needed - ".*" was for regex matching. Thanks for spotting it. I'll add a commit soon.